Updated the Getting Started section in the README.md file. (#420)
joschahenningsen/TUM-Live: README.md

```

### Setup Database
- Follow the steps [here](https://mariadb.com/kb/en/installing-and-using-mariadb-via-docker/) to install mariadb via docker.
- Then run the docker container using the following command.
```bash

> docker run --detach --name mariadb-tumlive --env MARIADB_USER=root --env MARIADB_PASSWORD=root --env MARIADB_ROOT_PASSWORD=example --restart always -p 3306:3306 mariadb:latest`

Reviewer: Is `--env MARIADB_PASSWORD=root` required?

Author: It is not required indeed, I found the list of available variables [here](https://mariadb.com/kb/en/mariadb-docker-environment-variables/)

Author: Removed it!

```
- Alternatively, install mariadb on its own.
- Create the database `tumlive` using [this]([tum-live-starter.zip](https://github.com/joschahenningsen/TUM-Live/files/8505487/tum-live-starter.zip)) script.

> - Optional: Use [JetBrains DataGrip](https://www.jetbrains.com/datagrip/) to open this project and then run the script there to automatically set up a demo database.

Reviewer: ```suggestion
- Or: Use [JetBrains DataGrip](https://www.jetbrains.com/datagrip/) to open the database and then run the script there to automatically set up a demo database.
```

### Installing go

> - Install **go 1.18** by following the steps [here](https://go.dev/doc/install)

Reviewer: ```suggestion
- Install **go >=1.18** by following the steps [here](https://go.dev/doc/install)
```

> - Preferably use [JetBrains GoLand](https://www.jetbrains.com/go/) and open this project as it simplifies this entire process

Reviewer: Can you add `Go to File -> Settings -> Go -> Go Modules and enable go modules integration`?

Author: Done!

- Run `npm i` in the `./web` directory to install the required node modules
- Run `go get ./...` to install the required go modules

> - If you want to customize the environment variables (for example mariadb username and password), copy the `config.yaml` file over to your home directory

Reviewer: ```suggestion
- If you want to customize the configuration (for example mariadb username and password), copy the `config.yaml` file over to `$HOME/.TUM-Live/config.yaml` and make your changes there to prevent accidentally committing them.
```

- Start the app by building and running `./cmd/tumlive/tumlive.go`

> - Head over to `http://localhost:8081` in your browser of choice and confirm that the page has loaded without any problems.

Reviewer: Somewhere we should mention that `npm run build-dev` in `./web` should be used when coding to automatically rebuild the frontend code when making changes.

Author: I added this.

- Voilà! Happy coding! :sparkles:

### Enable pre-commit hooks

- Make sure you have [staticcheck](https://staticcheck.io/docs/getting-started/)
and [pre-commit](https://pre-commit.com/#install) installed. If you have `pip` installed on your machine, you can install them with the following command
```bash

> go install honnef.co/go/tools/cmd/staticcheck@latest & pip install pre-commit

Reviewer: So the outputs don't overlap:
```suggestion
go install honnef.co/go/tools/cmd/staticcheck@latest && pip install pre-commit
```
